src/main/java/net/dv8tion/jda/api/interactions/commands/build/OptionData.java

```diff
@@ -32,6 +33,36 @@
  */
 public class OptionData implements SerializableData
 {
+    /**
+     * The highest positive amount Discord allows the {@link OptionType#NUMBER NUMBER} type to be.
+     */
+    public static final double MAX_POSITIVE_NUMBER = (1L << 55);
+    
+    /**
+     * The smallest negative amount Discord allows the {@link OptionType#NUMBER NUMBER} type to be.
+     */
+    public static final double MIN_NEGATIVE_NUMBER = -(1L << 55);
+
+    /**
+     * The highest length the name of an option can be.
+     */
+    public static final int MAX_NAME_LENGTH = 32;
+
+    /**
+     * The highest length the description of an option can be.
+     */
+    public static final int MAX_DESCRIPTION_LENGTH = 100;
+
+    /**
+     * The highest length the {@link #addChoice(String, String) String value} of a Choice can be.
```

Comment:
I question this description and variable name.
Isn't this actually `MAX_CHOICE_VALUE_LENGTH` ?

I'd update the variable name and docs accordingly

Reply:
I realize now that it is named this way because the others also don't prefix the fact that they are for choices 🤔

Reply:
I made corrections to the docs and renamed the var.

If you have further suggestions, maybe provide an example for a good description, because I'm not quite sure here...

Also, if we wanted to be really accurate would this probs need to be named `MAX_CHOICE_STRING_VALUE_LENGTH` or something...